### DISCUSSION TITLE
How to preserve the data between individual job executions

### DISCUSSION THREAD

**Discussion by adarshasp87** [jobrunr/jobrunr#208] [Q&A] [answered]
I have a requirement of persisting a few metadata in the context and use them in the next execution. According to my observations, jobs configured to run recurrently are stateless in nature. I wanted to know if there is an option to preserve data between individual job executions?

**Comment by rdehuyss** [comment, marked as answer]
I'm afraid this is currently not possible. I need to think if I will add it as a feature.

For now, I would use an external datastore of your choice.
